### ISSUE TITLE
Error on full-deploy gulp task

### ISSUE THREAD

**Issue by pjatx** [Pixel2HTML/shopify-skeleton#6]
Steps to Reproduce: 
1. Setup project
2. Run `gulp` or `gulp full-deploy`

Expected: 
Gulp tasks run without error

Actual:
```
[17:38:26] Using gulpfile ~/projects/dr-vims-staging/gulpfile.js
[17:38:26] Starting 'full-deploy'...
[17:38:26] Starting 'build'...
[17:38:26] Starting 'clean'...
[17:38:27] Finished 'clean' after 7.31 ms
[17:38:27] Starting 'fonts'...
[17:38:27] Finished 'fonts' after 11 ms
[17:38:27] Starting 'icons'...
[17:38:27] Finished 'icons' after 253 ms
[17:38:27] Starting 'images'...
[17:38:27] Finished 'images' after 6.94 ms
[17:38:27] Starting 'scripts'...
[17:38:27] Starting 'vendor:scripts'...
[17:38:27] Starting 'main:scripts'...
[17:38:28] 'vendor:scripts' errored after 748 ms
[17:38:28] Error: premature close
    at Pumpify.onclose (/Users/philipjohnson/projects/dr-vims-staging/node_modules/end-of-stream/index.js:47:67)
    at emitNone (events.js:105:13)
    at Pumpify.emit (events.js:207:7)
    at Pumpify.Duplexify._destroy (/Users/philipjohnson/projects/dr-vims-staging/node_modules/duplexify/index.js:203:8)
    at /Users/philipjohnson/projects/dr-vims-staging/node_modules/duplexify/index.js:179:10
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
[17:38:28] 'scripts' errored after 749 ms
[17:38:28] 'build' errored after 1.03 s
[17:38:28] 'full-deploy' errored after 1.03 s
[17:38:28] The following tasks did not complete: main:scripts
[17:38:28] Did you forget to signal async completion?
```

**Comment by shelldandy** [comment]
your vendor scripts have something wrong 😢can't help you there. its your JS in the end

**Comment by pjatx** [comment]
This was on a brand new install / setup of the project without adding anything new. Will setup a brand new store and try to repro

**Comment by shelldandy** [comment]
Make sure to install the new namescoped package:

```
npm install -g @pixel2html/generator-shopify
```

```
$ yo @pixel2html/shopify
```

**Comment by pjatx** [comment]
Just tried reinstalling according to instructions above and getting a new error : (

```
philipjohnson@Philips-MBP-3  ~/projects  mkdir p2html-test
 philipjohnson@Philips-MBP-3  ~/projects  cd p2html-test
 philipjohnson@Philips-MBP-3  ~/projects/p2html-test  npm install -g @pixel2html/generator-shopify
+ @pixel2html/generator-shopify@3.0.2
added 207 packages in 6.509s
 philipjohnson@Philips-MBP-3  ~/projects/p2html-test  yo @pixel2html/shopify
 ________________________________________
< A few general purpose questions now... >
 ----------------------------------------
        \    ,-^-.
         \   !oYo!
          \ /./=\.\______
               ##        )\/\
                ||-----w||
                ||      ||

               Cowth Vader
? Name of your Project? Test
 ______________________________________________________________
( To set up a shop now you will need to have                   )
( private API credentials at hand see more here:               )
( https://help.shopify.com/api/getting-started/api-credentials )
 --------------------------------------------------------------
   o
    o

           _                _
          / /.           _-//
         / ///         _-   /
        //_-//=========     /
      _///        //_ ||   ./
    _|                 -__-||
   |  __              - \   \
  |  |#-       _-|_           |
  |            |#|||       _   |
 |  _==_                       ||
- ==|.=.=|_ =                  |
|  |-|-  ___                  |
|    --__   _                /
||     ===                  |
 |                     _. //
  ||_         __-   _-  _|
     \_______/  ___/  _|
                   --*
? Do you want to set up your shop credentials now? Yes
? Shop Name (Its the part that goes before the .myshopify.com bit on the admin) xxxxxxxxxxxxxxxxxxxx
? API Key
 philipjohnson@Philips-MBP-3  ~/projects/p2html-test  cd ..
 philipjohnson@Philips-MBP-3  ~/projects  sudo rm -r p2html-test
 philipjohnson@Philips-MBP-3  ~/projects  mkdir p2h-test
 philipjohnson@Philips-MBP-3  ~/projects  cd p2h-test
 philipjohnson@Philips-MBP-3  ~/projects/p2h-test  yo @pixel2html/shopify
 ________________________________________
< A few general purpose questions now... >
 ----------------------------------------
        \    ,-^-.
         \   !oYo!
          \ /./=\.\______
               ##        )\/\
                ||-----w||
                ||      ||

               Cowth Vader
? Name of your Project? Test
 ______________________________________________________________
( To set up a shop now you will need to have                   )
( private API credentials at hand see more here:               )
( https://help.shopify.com/api/getting-started/api-credentials )
 --------------------------------------------------------------
   o
    o

           _                _
          / /.           _-//
         / ///         _-   /
        //_-//=========     /
      _///        //_ ||   ./
    _|                 -__-||
   |  __              - \   \
  |  |#-       _-|_           |
  |            |#|||       _   |
 |  _==_                       ||
- ==|.=.=|_ =                  |
|  |-|-  ___                  |
|    --__   _                /
||     ===                  |
 |                     _. //
  ||_         __-   _-  _|
     \_______/  ___/  _|
                   --*
? Do you want to set up your shop credentials now? Yes
? Shop Name (Its the part that goes before the .myshopify.com bit on the admin) xxxxxxxxxxxxx
? API Key xxxxxxxxxxxxxxx
? Password xxxxxxxxxxxx
? Shared Secret xxxxxxxxxxxxxxx
? Theme ID (Create a new theme click customize HTML/CSS and its after the themes/ part on the URL) xxxxxxxxxxxxxxx
   create package.json
   create src/theme/config/settings_schema.json
   create src/theme/layout/checkout.liquid
   create src/theme/layout/theme.liquid
   create src/theme/locales/en.default.json
   create src/theme/locales/es.json
   create src/theme/sections/featured-collection.liquid
   create src/theme/snippets/svg-icons.liquid
   create src/theme/templates/customers/account.liquid
   create src/theme/templates/customers/activate_account.liquid
   create src/theme/templates/customers/addresses.liquid
   create src/theme/templates/customers/login.liquid
   create src/theme/templates/customers/order.liquid
   create src/theme/templates/customers/register.liquid
   create src/theme/templates/customers/reset_password.liquid
   create src/theme/templates/404.liquid
   create src/theme/templates/article.liquid
   create src/theme/templates/blog.liquid
   create src/theme/templates/cart.liquid
   create src/theme/templates/collection.liquid
   create src/theme/templates/gift_card.liquid
   create src/theme/templates/index.liquid
   create src/theme/templates/list-collections.liquid
   create src/theme/templates/page.contact.liquid
   create src/theme/templates/page.liquid
   create src/theme/templates/password.liquid
   create src/theme/templates/product.liquid
   create src/theme/templates/search.liquid
   create .env
   create .env.example
   create .gitignore
   create LICENSE
   create editorconfig
   create gulpfile.js
   create readme.md
   create webpack.config.js
   create gulp/gulp.config.js
   create gulp/tasks/clean.js
   create gulp/tasks/deploy.js
   create gulp/tasks/fonts.js
   create gulp/tasks/icons.js
   create gulp/tasks/images.js
   create gulp/tasks/scripts.js
   create gulp/tasks/serve.js
   create gulp/tasks/shopify.js
   create gulp/tasks/styles.js
   create gulp/tasks/watch.js
   create gulp/tasks/zip.js
   create src/fonts/.gitkeep
   create src/icons/gulp.svg
   create src/images/shopify.svg
   create src/scripts/index.js
   create src/styles/main.scss
   create src/styles/vendor.scss
+--------------------------+
|Thank you and Good Luck! 🦄|
+--------------------------+
       \
        \     ▄▄▄▄▄▄▄▄
         \  ▄ ▄      ▄▄ ▄
          ▄ ▄            ▄
          ▄      ▄▄▄▄▄
         ▄               ▄ ▄
        ▄
                 ▄
                  ▄ ▄
         ▄        ▄ ▄      ▄▄ ▄
          ▄                   ▄▄ ▄
          ▄ ▄▄▄▄                 ▄▄ ▄
              ▄▄▄▄                  ▄▄
        ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀



PS: use npm run code to start
Love, Pixel2HTML
 philipjohnson@Philips-MBP-3  ~/projects/p2h-test  npm run code

> Test@1.0.0 code /Users/philipjohnson/projects/p2h-test
> cross-env NODE_ENV=development gulp

sh: cross-env: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! Test@1.0.0 code: `cross-env NODE_ENV=development gulp`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the Test@1.0.0 code script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/philipjohnson/.npm/_logs/2018-01-12T23_51_27_720Z-debug.log
 ✘ philipjohnson@Philips-MBP-3  ~/projects/p2h-test  npm install
npm WARN deprecated minimatch@0.2.12: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated node-uuid@1.4.0: Use uuid module instead
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! code ETARGET
npm ERR! notarget No matching version found for stylelint-config-pixel2html@^0.1.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'Test'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/philipjohnson/.npm/_logs/2018-01-12T23_51_54_541Z-debug.log
```

**Comment by shelldandy** [comment]
Thanks for the report I'll look into it in more depth next week. I gotta update some dependencies and run some more tests.

**Comment by shelldandy** [comment]
I've edited your comment to remove the API crendentials :v

**Comment by pjatx** [comment]
Thanks! 

What's the latest stable version I can use in the mean time?

**Comment by shelldandy** [comment]
Any v 2.XX

**Comment by pjatx** [comment]
ha - thanks! it's just a blank store, so feel free to play with it. just downloaded the debut theme and reuploaded it

**Comment by shelldandy** [comment]
```
npm i -g yo generator-pixel2html-shopify@2.1.1
$ yo pixel2html-shopify
```

**Comment by shelldandy** [comment]
Also take a look here, its unmaintained and a bit lonely but the webpack setup is super cool.

https://github.com/DynamoMTL/shopify-pipeline

I might use some of that tech in here in the future...

**Comment by shelldandy** [comment]
Fixed on 3.0.2

```
npm i -g yo @pixel2html/shop-generator
```
